model/testing/src/icon4py/model/testing/helpers.py

```diff
@@ -77,6 +77,7 @@ def allocate_data(backend, input_data):
 def apply_markers(
     markers: tuple[pytest.Mark | pytest.MarkDecorator, ...],
     backend: gtx_backend.Backend | None,
+    has_skip_values: bool = False,
```

Comment:
Or just pass the grid here? The kind of grid might be needed anyway at some point for selecting tests.

Reply:
yes, I agree.  I did that before but the function is also called in the `pytest_runtest_setup`, I need to check whether I can access the session scoped fixtures there. (probably yes somehow...)